SYCL/Reduction/reduction_usm_dw.cpp

```diff
@@ -35,7 +31,7 @@ int test(queue &Q, T Identity, T Init, size_t WGSize, size_t NWItems,
     return 0;
   if (AllocType == usm::alloc::device) {
     Q.submit([&](handler &CGH) {
-       CGH.single_task<KName<Name, class InitKernel>>(
+       CGH.single_task<USMKName<Name, class InitKernel>>(
```

Comment:
```suggestion
       CGH.single_task(
```

Reply:
Can't do - crashes in the RT inside program_manager.cpp. Will try to follow-up later.